src/components/explore/QuantileExplorerView.svelte

```diff
@@ -32,10 +33,10 @@
   let aggregationTypes = gatherAggregationTypes(data);
   let probeKeys = gatherProbeKeys(data);
 
-  let currentKey = probeKeys[0];
+  let currentKey = $store.aggKey || probeKeys[0];
   let currentAggregation = aggregationTypes.includes('summed_histogram')
     ? 'summed_histogram'
-    : 'avg';
+    : $store.aggType;
```

Comment:
If there isn't one stored does it still default to `'avg'`?

Reply:
It should still default to 'avg'. See: https://github.com/mozilla/glam/pull/1531/files#diff-352b8684efbebfa2b2a01b4af9892eea7fb1c2f2f9cd6c4acf803c7fa8974c32R40